src/libecalc/core/models/compressor/train/base.py

```diff
@@ -138,6 +139,27 @@ def evaluate_rate_ps_pd(
             failure_status=[t.failure_status for t in train_results],
         )
 
+    def evaluate_streams(
+        self,
+        inlet_streams: List[Stream],
+        outlet_stream: Stream,
+    ) -> CompressorTrainResult:
+        """
+        Evaluate model based on inlet streams and the expected outlet stream.
+        Args:
+            inlet_streams:
+            outlet_stream:
+
+        Returns:
+
+        """
+        mixed_inlet_stream = Stream.mix_all(inlet_streams)
```

Comment:
Do we ever need all input streams here? Maybe it is better to mix before we evaluate? Then always treat 1 in and output stream here?

Reply:
We should consider it, we could also look at the difference between the Streams container and operational_settings passed to compressor and pump. We could only pass Streams, then add separate arguments to evaluate for pump and compressor if we need to pass something else than streams.

Reply:
A compressor train could use the streams separately, so that is the point of this PR, to give that option when creating a model. Multiple streams does that. Currently we don't have an id for streams, so index must be used, but I think we should look into adding that id (name in yaml, from user?)